### ISSUE TITLE
Explore impact of full capitalisation when used in destructive action buttons

### ISSUE THREAD

**Issue by DBD324** [mi6/ic-ui-kit#192]
## Summary
Destructive action buttons use full capitalisation for their labels. This was initially intended to help add emphasis to the destructive action and force the user to slow down and take time to comprehend the action they are about to conduct. However, the effect of capitalisation on readability can be significant. Need to research in detail the trade-off between readability and emphasis in this case and potentially explore other methods of emphasis. 

Additionally, the content style section also mentions not using capitalisation due to the readability issues. If valid edge cases exist, then this content should be updated to reflect. 

## 💬 Description
Research the effect of the capitalisation on readability of short terms. 
Review content style to reflect the research on this issue and capture edge cases. 

## 💰 Use value
This is important as it could help to improve the usability of the site for users who may find it difficult to comprehend terms written in all-caps. It will help to back up the decision with links to research (primary or secondary).

## Additional info
Some initial secondary research:
- https://uxplanet.org/why-letter-casing-is-important-to-consider-during-design-decisions-50402acd0a4e
- https://learn.microsoft.com/en-us/typography/develop/word-recognition
- https://www.blog.theteamw.com/2009/12/23/100-things-you-should-know-about-people-19-its-a-myth-that-all-capital-letters-are-inherently-harder-to-read/
- https://www.nngroup.com/articles/glanceable-fonts/
-

**Comment by GCHQ-Developer-094** [comment]
Won't do for now
